### PR TITLE
ISPN-5542 Shorten interceptor stack for local caches

### PR DESCRIPTION
**Pull request by rvansa**
* fixed javadoc, XSD and moved the attributes to Parser80 instead of Parser72

### CODE REVIEW

File: core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java [hunk 1]
```diff
@@ -9,6 +7,8 @@
 import org.infinispan.util.logging.Log;
 import org.infinispan.util.logging.LogFactory;
 
+import java.lang.invoke.MethodHandles;
```
Comment (Sanne) [inline core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java]: you should stop this :)

Comment (rvansa) [inline core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java]: :-/ I have to set the settings in one way before commit, and IntelliJ IDEA automatically formats all modified files. So I would either change the imports order in Parser72/80 or here.

Comment (rvansa) [inline core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java]: OK, unchecking 'optimize imports on the fly' should help, hopefully...

Comment (Sanne) [inline core/src/main/java/org/infinispan/configuration/cache/JMXStatisticsConfigurationBuilder.java]: thanks!

**Comment by tristantarrant** [comment]
We need server configuration to stay symmetric with embedded, so please add it to the subsystem as well

**Comment by rvansa** [comment]
@tristantarrant Thanks for reminding, I have added those. Though, I am not sure whether notifications should really be in server configuration since you can't register listeners on server anyway, but I've included those to keep the config symmetric.

**Comment by galderz** [comment]
Integrating, please format commit messages appropiately (see [here](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))

**Comment by galderz** [comment]
Integrated after shorting out format (add a empty line between title and comments)

**Pull request closed without merging**